internal/support/logging/init.go

```diff
@@ -86,6 +80,7 @@ func (s ServiceInit) BootstrapHandler(
 	}
 
 	// initialize database.
+	var dbClient interfaces.Logger
```

Comment:
I wonder if should name this something different, something like `persistenceClient`, since this can be something other than a database. Just a thought.

Reply:
Created https://github.com/edgexfoundry/edgex-go/issues/2211